salt/apps/ocw/symlinks_cms.sls

```diff
@@ -0,0 +1,82 @@
+
+symlink_prod_new:
+  file.symlink:
+    - name: /prod_new
+    - target: /mnt/ocwfileshare
+    - force: True
+    - user: root
+    - group: root
+
+symlink_prod:
+  file.symlink:
+    - name: /prod
+    - target: /prod_new
```

Comment:
Why are we symlinking to a symlink?

Reply:
Not that any of this makes a lot of sense, but I was following the library paradigm -- for example in some systems' `/usr/lib` -- where there are layers of symlinks from more general to more specific. But it's not really necessary here, I acknowledge. I'll change it.